<?xml version="1.0"?>

<launch>

	<node pkg="lns_navigation" type="gps_waypoint" name="gps_waypoint" output="screen"/>
	<param name="coordinates_file" value="/waypoint_files/points_sim.txt" type="string" />	
	
	
</launch>
